utils/helpers/constants/token-type.js

```diff
@@ -0,0 +1,6 @@
+import { TokenSupplyType } from "@hashgraph/sdk";
```

Comment:
This should be TokenType, also please check why the "**Token Type**" test suite is passing.

Reply:
I was trying to restrict use of the JS SDK as much as I could, I feel like since this is meant to test the JS SDK it should keep its dependence on it to a minimum. We could use another enum to track `"nft"` and `"ft"` token types, but the JSON sent across is a string so it must be one of these two options

Reply:
I agree that avoiding the use of the JS SDK and the TokenType class is the better approach. However, based on my personal experience, it might be a bit confusing for a new team member who has previously used TokenType to create a TokenCreateTransaction. Since it can be stringified using .toString(), it could still be used in this context. That said, I understand that simplifying things and steering clear of direct JS SDK usage will allow us to focus more on the tests rather than on the SDK itself.

In short, let's stick with the current approach.